frontend/sw360-portlet/src/main/resources/content/Language_vi.properties

```diff
@@ -1087,7 +1087,7 @@ used.in=Được dùng trong
 used.license=Giấy phép được sử dụng
 user=Người dùng
 user.cannot.edit.a.closed.project=Người dùng không thể chỉnh sửa một dự án đã đóng
-UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360
+UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Clearing Expert: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360
```

Comment:
May be We should add the Vietnamese version of `Clearing Expert:` here.

Reply:
Hi, thank you for your notification. 
Could you help me add Vietnamese in this line for "Clearing Expert":
`UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Chuyên gia xóa: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360`

Tks

Reply:
@nam-np Thanks for sharing the translation.
@smrutis1 I have added the Vietnamese translation of `Clearing Expert`.